fossunited/api/schedule.py

```
@@ -52,129 +44,76 @@ def safe_speaker_name(sp):
-    """
 
-    if local.request:
-        days_list = local.request.args.getlist("days")
-        halls_list = local.request.args.getlist("halls")
-    else:
-        # fallback: parse comma-separated string params if provided
-        days_list = days.split(",") if days else []
-        halls_list = halls.split(",") if halls else []
+# nosemgrep: guest-whitelisted-method
+@frappe.whitelist(allow_guest=True)
+def download_schedule(event, format="ics", days="", halls=""):
```

Ruff (0.15.9)

[error] Function argument `format` is shadowing a Python builtin

(A002)